artifactory_test.go

```diff
@@ -4826,6 +4826,35 @@ func TestConfigAddOverwrite(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestConfigConcurrency(t *testing.T) {
```

Comment:
I don't think concurrency tests should be included in the jfrog-cli code base. The tests should be deterministic. Having such tests running automatically may produce unexpected results. Let's remove this one.
So besides this test, which is current commented out, I'm actually not sure what this PR adds.